src/openforms/formio/registry.py

```diff
@@ -91,7 +91,10 @@ def build_serializer_field(self, component: ComponentT) -> serializers.Field:
             DeprecationWarning,
         )
 
-        if is_layout_component(component):
+        # not considered a layout component (because it doesn't have children)
+        if component["type"] == "content":
+            required = False
```

Comment:
Probably at some point if we manage to define components as dataclass-likes structure, this will be easier to handle (via a method/property)

Reply:
yeah, this patch definitely had me longing for such a thing again :)